documentation/book/cluster-operator.adoc

```diff
@@ -414,26 +418,30 @@ The same syntax as Kubernetes' is used for the values of `cpu` and `memory`.
 }
 ----
 
+:k8s-docs-version: v1-7
+:k8s-resource-request-limit-docs-link: https://{k8s-docs-version}.docs.kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+
 `requests/memory`::
-the memory request for the container, corresponding directly to Kubernetes' {resources-link}[`spec.containers[\].resources.requests.memory`] setting.
-Kubernetes/OpenShift will ensure the containers have at least this much memory by running the pod on a node with at least as much free memory as all the containers require.
-Optional with no default.
+the memory request for the container, corresponding directly to {k8s-resource-request-limit-docs-link}[`spec.containers[\].resources.requests.memory`] setting.
+{ProductPlatformName} will ensure the containers have at least this much memory by running the pod on a node with at
+least as much free memory as all the containers require. Optional with no default.
```

Comment:
A week or two ago @mbogoevici linked to some style guide advocating a physical-line-per-sentence style. I know I'm usually the one complaining about long lines, but I've been trying to use that approach since then, and I've found I don't hate it. In any case we should be consistent, as rewrapping docs like this just ends up being noise. Can we reach a consensus?

Reply:
Sorry for this ... this might have been caused by the semi manual merging :-(. In general, I think one sentence per line requires short sentences to be comfortable.

Reply:
FWIW, [this was the style guide](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line). Short sentences are usually better than longer ones. And most editors can soft wrap long lines. In the week or two since I started doing this (for asciidoc only I should add) I've found that it's not that awful.

Reply:
If there is a consensus, I'm fine with this. I added to sync up agenda.

Reply:
I've been using one-sentence-per-line for a few years now in different settings. In particular on the Spring team, we've found that it massively improves the change review process, especially when dealing with minor updates and/or additional commits in a PR. It also encourages shorter sentences :).